Geometry/CMSCommonData/python/cmsIdealGeometry2015XML_cfi.py

```diff
@@ -18,6 +18,7 @@
         'Geometry/CMSCommonData/data/cmsBeam.xml',
         'Geometry/CMSCommonData/data/muonMB.xml',
         'Geometry/CMSCommonData/data/muonMagnet.xml',
+        'Geometry/TrackerCommonData/data/trackerParameters.xml',
```

Comment:
@pietverwilligen - this is part of a separate PR. I think, it is better to remove this file from here.